app/src/main/java/org/simple/clinic/datepicker/calendar/CalendarDatePicker.kt

```diff
@@ -5,9 +5,9 @@ import android.app.Dialog
 import android.content.Context
 import android.content.DialogInterface.BUTTON_NEGATIVE
 import android.os.Bundle
-import android.os.Parcelable
 import androidx.fragment.app.DialogFragment
 import kotlinx.android.parcel.Parcelize
+import org.simple.clinic.datepicker.SelectedDate
```

Comment:
This should have been added in the `CalendarDatePicker` commit for that commit to compile. Since we're squashing so this works as well.

Reply:
Actually, that was the reason, we have stopped trying to make sure every commit works. As long as the PR as whole works it's fine. We create smaller commits with specific changes so that its' easier to review.

Reply:
Also in this scenario, it will compile. The `SelectedDate` class used here is from the key factory and the one we had previously was present in this class already. So it shouldn't have any compilation errors 🤔